pkg/action/hooks.go

```diff
@@ -73,7 +73,7 @@ func (cfg *Configuration) execHook(rl *release.Release, hook release.HookEvent,
 		h.LastRun.Phase = release.HookPhaseUnknown
 
 		// Create hook resources
-		if _, err := cfg.KubeClient.Create(resources); err != nil {
+		if _, err := cfg.KubeClient.Create(resources, false); err != nil {
```

Comment:
It seems like (for consistency) the operations `DisableOpenAPIValidation` value should be passed in here.

Otherwise, doing e.g. `helm install --disable-openapi-validation` could still fail schema validation for a malformed hook

(that said, this PR is fixing a regression. it doesn't need to be perfect to be an improvement)